content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -56,12 +56,22 @@ endpoint** in your **Hosted endpoints**. A form opens, where you'll need to add
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+.. _stripe/pmt:
```

Comment:
I'd rather use meaningful words rather than abbreviations (it also helps the SEO... I think ^^)

```suggestion
.. _stripe/local-payment-methods:
```